test/components/ColumnCard.nuxt.test.ts

```diff
@@ -0,0 +1,35 @@
+import { screen } from '@testing-library/vue'
+import { renderSuspended } from '@nuxt/test-utils/runtime'
+import { ColumnCard } from '#components'
+import { taskTruncatedIdLength } from '~/constants/tasks'
+
+describe('<ColumnCard />', () => {
+  it('should render the title (link to a new tab), status, id (truncated), and labels', async () => {
```

Comment:
Just saw this. If we're using the `it()` assertion, then the test description shouldn't use the "should" word in it. That's an alternative notation `should()`. Either way, the string inside those should be in the present tense and as concise as possible (think formal spec/design document). In this case, I'd personally go with:

```suggestion
  it('renders the title (link as new tab), status, id (truncated), and labels', async () => {
```

The above also negates the [need for the linting rule change](https://github.com/cytechmobile/radicle-planning-boards/pull/73#discussion_r1636204954) to begin with. At least for this one case, but it rules are there for a reason. If the description of a test case is too long it may be able to be worded more concisely, it may be doing too much and if all else fail then we should shamefully `// eslint-disable-next-line max-len`.

Reply:
👍 Updated as suggested